scrubadub/detectors/base.py

```diff
@@ -138,7 +138,10 @@ def iter_filth(self, text: str, document_name: Optional[str] = None) -> Generato
             raise ValueError('No regular expression has been specified for {}.'.format(self.__class__))
 
         for match in self.regex.finditer(text):
-            yield self.filth_cls(match=match, detector_name=self.name, document_name=document_name, locale=self.locale)
+            filth = self.filth_cls(match=match, detector_name=self.name, document_name=document_name, locale=self.locale)
```

Comment:
this is causing a PEP8 error for line length

Reply:
fixed. thanks!